papertty/drivers/driver_it8951.py

```diff
@@ -228,6 +228,10 @@ def init(self, **kwargs):
         # Set to Enable I80 Packed mode.
         self.write_register(self.REG_I80CPCR, 0x0001)
 
+        for key, value in kwargs.items():
+            if key == 'vcom':
+                self.VCOM = value
```

Comment:
This could be made a bit more concise:
```suggestion
        self.VCOM = kwargs.get('vcom', self.VCOM)
```